keyboards/sanctified/dystopia/info.json

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
Please redo this file without the rotation, it causes the Configurator to render this instead:
![image](https://user-images.githubusercontent.com/4781841/109406930-07262c00-79d1-11eb-8342-d7d4fd61d09f.png)

Reply:
Thank you for pointing this out, I've updated the file for it to display better with commit 17a8ca3